Marlin/configuration_store.cpp

```diff
@@ -324,9 +344,9 @@ void Config_StoreSettings()  {
     EEPROM_WRITE_VAR(i, dummy);
   }
 
-  char ver2[4] = EEPROM_VERSION;
```

Comment:
// Now store checksum in stored_checksum variable
uint16_t stored_checksum = eeprom_checksum;